rust/otap-dataflow/crates/otap/src/otap_grpc/client_settings.rs

```diff
@@ -199,9 +271,16 @@ const fn default_keep_alive_while_idle() -> bool {
 }
 
 #[cfg(test)]
+#[allow(missing_docs)]
 mod tests {
     use super::*;
 
+    #[cfg(feature = "experimental-tls")]
```

Comment:
I have a sense that users will always want TLS and that it might cost us in development velocity to maintain it as a feature. Should we include TLS immediately? (I think yes!)